src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -16,18 +16,217 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
-    internal sealed class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
+    internal class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
```

Comment:
Since removing `sealed` on the type, most of the members should now be marked as sealed instead.

Reply:
i tried to do just that :)

Reply:
done.